packages/vanilla/src/components/footer/README.md

```diff
@@ -0,0 +1,119 @@
+# Footer
+
+> The footer component is a required navigational pattern for IBM.com that
+> displays consistently at the bottom of each page.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import { Footer } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+import '@carbon/ibmdotcom-styles/scss/components/footer/index.scss';
+
+console.log(
+  footer({
+    type: '', // tall | short
+  })
+);
+```
+
+> 💡 Don't forget to import the footer styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+### Initialize
+
+> Init example below
+
+```javascript
+import { Accordion } from 'carbon-components';
+import { globalInit } from '@carbon/ibmdotcom-services';
+
+class Footer {
+  static init(El) {
+    globalInit();
+
+    if (El) {
+      Accordion.create(El);
+    }
+  }
+}
+```
+
+#### Feature Flags
+
+To utilize the following features, set the following variable's to `true` within
+your `.env` file or your application build settings.
+
+```
+FOOTER_LOCALE_BUTTON=true
+```
+
+> See
+> [feature-flags.md](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/docs/feature-flags.md)
+> and
+> [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/.env.example)
+> for more information
+
+## Types (optional)
```

Comment:
@andysherman2121 so @annawen1 just updated a bunch of READMEs, can you doublecheck the latest on `master`?